src/solvers/dgmulti/flux_differencing.jl

```diff
@@ -193,20 +187,27 @@ function create_cache(mesh::VertexMappedMesh, equations, dg::DGMultiFluxDiff{<:P
             local_values_threaded, fluxdiff_local_threaded, rhs_local_threaded)
 end
 
+# TODO: DGMulti. Address hard-coding of `entropy2cons!` and `cons2entropy!` for this function.
 function entropy_projection!(cache, u, mesh::VertexMappedMesh, equations, dg::DGMulti)
 
   rd = dg.basis
   @unpack Vq = rd
   @unpack VhP, entropy_var_values, u_values, entropy_var_values = cache
   @unpack projected_entropy_var_values, entropy_projected_u_values = cache
 
-  # TODO: simplices. Address hard-coding of `entropy2cons!`
   apply_to_each_field(mul_by!(Vq), u_values, u)
-  entropy_var_values .= cons2entropy.(u_values, equations)
+
+  # TODO: DGMulti. @threaded crashes when using `eachindex(u_values)`.
+  # See https://github.com/JuliaSIMD/Polyester.jl/issues/37 for more details.
+  for i in Base.OneTo(length(u_values))
```

Comment:
```suggestion
  # TODO: DGMulti. @threaded crashes when using `eachindex(u_values)`.
  # See https://github.com/JuliaSIMD/Polyester.jl/issues/37 for more details.
  @threaded for i in Base.OneTo(length(u_values))
```
But you're not using `eachindex` anymore, so adding `@threaded` here should be fine, shouldn't it?

Reply:
Oops - yes! I accidentally committed my experimentation...